### ISSUE TITLE
Converting to TensorRT 5

### ISSUE THREAD

**Issue by Spectra456** [matterport/Mask_RCNN#1662]
If anybody successfully converted this mask-rcnn model to tensorRT model?

**Comment by i-chaochen** [comment]
Yes, I am looking the same thing! but I got the strange outputs when I used the frozen graph of this mask-rcnn.  https://github.com/matterport/Mask_RCNN/issues/1668

So I am not sure it still works well when you converted to TensorRT, as it will do some pruning and optimization.

Let me know if you're done it.  Thanks!

**Comment by hwh-hit** [comment]
@Spectra456 @i-chaochen using .pb file, i got the exactly same result with the original .h5 file, when you convert h5 to pb, you must keep the exactly same params. when i converted it to trt, there are a lot of unsupported ops ( tf.cast tf.where tf_equal.......). maybe we should write custom layers to implement those ops ，but i haven't finished that.
